### ISSUE TITLE
Questions about training datasets

### ISSUE THREAD

**Issue by GuiyuZhao** [qinzheng93/GraphSCNet#3]
Hi! a very good work! I have a question about the dataset you used. In your code's train_valid_data_loader, are you using the "val" folder as the training set? 4DMatch-F" folder as a test set? So the data in the "train" folder is not used？

**Comment by qinzheng93** [comment]
Yes. The "train" split has been used to train the correspondence extractor, so there is a large bias between the "train" and other splits. To eliminate this bias, we use "val" for training the outlier rejection model. This has been described in the paper.
